pom.xml

```diff
@@ -149,6 +150,16 @@
         <artifactId>api-common</artifactId>
         <version>${google.api-common.version}</version>
       </dependency>
+      <dependency>
+        <groupId>com.google.cloud</groupId>
+        <artifactId>google-cloud-iamcredentials</artifactId>
+        <version>0.43.0-alpha</version>
```

Comment:
Declaring this version inside `<properties>` tag would be better.

Reply:
done